src/posts/interning-at-khan-academy.md

```diff
@@ -0,0 +1,51 @@
+title: "Interning at Khan Academy: from student to intern"
+published_on: November 19, 2016
+author: Shadaj Laddad
+team: Web Frontend
+...
+
+This past summer, I had the amazing opportunity to work on a product I use as a student every day -- Khan Academy. During my internship, I worked on the web frontend team on the XOM (x-on-mobile) initiative to bring Khan Academy exercises to mobile devices. In this blog post, I’ll be going through a highly-compressed summary of my two months at Khan Academy, but since most of my work here was in open-source projects you can check out my work in detail on GitHub ([https://github.com/Khan/perseus/commits?author=shadaj](https://github.com/Khan/perseus/commits?author=shadaj), [https://github.com/Khan/math-input/commits?author=shadaj](https://github.com/Khan/math-input/commits?author=shadaj)).
+
+# Part 1: ramping up
+
+My first day at Khan Academy was a whirlwind of activities. From setting up my laptop to having a 1:1 with my mentor and finally adding myself to the interns page, I had a lot of things to do. As soon as I settled down with my first deploy, I got to work on the XOM initiative, starting with improving the math keypad.
+
+As part of the XOM initiative, the team had designed a math keypad that could be used to type in expression responses to mathematics problems in a mobile-friendly way. I started by fixing some bugs to get familiar with the math keypad codebase, such as disabling gestures on the static sidebar. Once I had a good idea of how the keypad worked, I was able to start working on adding new features our designers had crafted. I worked on things such as using square icons for representing empty blocks, and learned in the process how important it is to make these small changes to create a great experience for students.
+
+At Khan Academy, we use React.js, something I had worked with before, but only in the context of [Scala.js](https://www.scala-js.org) applications. As part of making changes to the math keypad, I had to learn how to write idiomatic ES6 code. Working on Khan Academy gave me a great opportunity to see use of React with ES6 in a production environment, and left me with many ideas on how React.js could be used more seamlessly with Scala.js. In addition to learning React and ES6, I got a chance to work with the plethora of open-source libraries at Khan Academy such as Aphrodite ([https://github.com/khan/aphrodite](https://github.com/khan/aphrodite)) and KaTeX ([https://github.com/khan/katex](https://github.com/khan/katex)).
```

Comment:
"to see use of React" is a bit awkward, maybe "to use React...".  Can you add brief descriptions of Aphrodite and Katex.  Also the words "Aphrodite" and "KaTeX" could be the link themselves.